examples/instrumentation-quickstart/src/main/resources/logback.xml

```diff
@@ -25,15 +25,16 @@
           {
             "logging.googleapis.com/trace": "%mdc{trace_id}",
             "logging.googleapis.com/spanId": "%mdc{span_id}",
-            "logging.googleapis.com/trace_sampled": "#asBoolean{%replace(%mdc{trace_flags}){'01', 'true'}}"
+            "logging.googleapis.com/trace_sampled": "#asBoolean{%replace(%mdc{trace_flags}){'01', 'true'}}",
+            "severity": "%replace(%replace(%level){'WARN', 'WARNING'}){'TRACE', 'DEBUG'}"
```

Comment:
@dashpole any thoughts on this? Do you find it to be decently readable or hacky?

Reply:
definitely seems hacky to me, but I would say the same about the trace_sampled line.  I'm not familiar with the operators (# or %)

Reply:
not sure if someone used to logback would be more familiar

Reply:
We are using the logstash encoder for JSON support which has a configurable [JSON provider](https://github.com/logfellow/logstash-logback-encoder?tab=readme-ov-file#pattern-json-provider.). The [conversion specifiers](https://logback.qos.ch/manual/layouts.html#conversionWord) are mostly provided by logback.

> not sure if someone used to logback would be more familiar

+1

Reply:
I think we should create a tracking ticket for this work,  I think it might require some dedicated effort to figure out the solution to this.

What might help in the meantime could be comments within the XML explaining the rationale behind this change because it is not straightforward for the reader.